docs/components/modeler/bpmn/business-rule-tasks/business-rule-tasks.md

```diff
@@ -9,24 +9,74 @@ modeled in [Decision Model and Notation](https://www.omg.org/dmn/) (DMN).
 
 ![task](assets/business-rule-task.png)
 
-Business rule tasks behave exactly like [service tasks](/components/modeler/bpmn/service-tasks/service-tasks.md). Both
-task types are based on jobs and [job workers](/components/concepts/job-workers.md).
+:::note
+Camunda Cloud supports alternative task implementations for the business rule task. If you want to
+use your own implementation for a business rule task, see the **Alternative task implementation**
```

Comment:
:wrench: We could use a link for `Alternative task implementation` to navigate directly to the section.

Reply:
I had this before, but I changed it after reading the [Writing Style Guide](https://confluence.camunda.com/display/HAN/Camunda+Writing+Style+Guide). It recommends bolding when referring to another section within a document.

Reply:
:thinking: I'm okay with it. But it doesn't say that it can't be a link :stuck_out_tongue: 
Personally, I find it more useful to click on a click when the text refers to a section.

Reply:
👍 I kinda agree with you on this one.

❓ @christinaausley Can you clarify your stance on this? Does the writing guide favor bolding over links when it comes to referring to other sections on the same page?

Reply:
Oh yes! A link is definitely preferred over bolding. I'll make sure this is more clear in the style guide -- thank you for looping me in!